target/public/.kbn-optimizer-cache

```diff
@@ -77,15 +77,15 @@
       "/Users/lijshu/Projects/7.9.1/kibana-workbench/packages/kbn-optimizer/target/worker/entry_point_creator.js": 1603134193348.825,
       "/Users/lijshu/Projects/7.9.1/kibana-workbench/packages/kbn-optimizer/target/worker/postcss.config.js": 1603134193349.5967,
       "/Users/lijshu/Projects/7.9.1/kibana-workbench/packages/kbn-ui-shared-deps/public_path_module_creator.js": 1601496201781.9238,
-      "/Users/lijshu/Projects/7.9.1/kibana-workbench/plugins/kibana-notebooks/common/index.ts": 1603136158017.4167,
```

Comment:
is this file necessary in the package? Looks like it's all the cache in your local environment

Reply:
This is an open [issue](https://github.com/elastic/kibana/issues/70426) from upstream kibana on new plugin platforms, `yarn build` is not generating these files and the artifact won't work. The only workaround I found so far is to push the target directory to git so it builds successfully